views/users/manage-users.handlebars

```diff
@@ -0,0 +1,43 @@
+<div>
+    USERS 
+    <span class="list-group-item-success">Enabled</span> 
+    <span class="list-group-item-danger">Disabled</span>
+    <span>★Admin★</span> 
+    
+    <br />
+    <code id="manage-error"></code>
+    
+    <br />
+    <ol class="list-group list-group-numbered pull-left">
+        {{#each users as |user| }}
+            <span>
```

Comment:
do you need the <span> for some reason?

Reply:
it was leftover from trying to push buttons to the right, but `pull-right` from bootstrap fixed many things!